operator/config/overlays/openshift/kustomization.yaml

```diff
@@ -31,60 +26,15 @@ labels:
     app.kubernetes.io/version: "0.0.1"
 
 patchesStrategicMerge:
-# Protect the /metrics endpoint by putting it behind auth.
-# If you want your controller-manager to expose the /metrics
-# endpoint w/o any authn/z, please comment the following line.
+- auth_proxy_service_annotations_patch.yaml
 - manager_auth_proxy_patch.yaml
 - manager_related_image_patch.yaml
 - manager_run_flags_patch.yaml
+- manager_webhook_patch.yaml
 - prometheus_service_monitor_patch.yaml
+- webhook_service_annotations_patch.yaml
 
-# apiVersion: kustomize.config.k8s.io/v1beta1
-# kind: Kustomization
```

Comment:
I am not sure I understand the ask here. I removed these lines because they were unused/obsolete before. What do I miss here? Was the issue already in place by having the commented?